### ISSUE TITLE
Continuous [integration/delivery/deployment]

### ISSUE THREAD

**Issue by lucasconstantino** [MarcelLZ/barbecue-company#81]
I've notices you are using Travis here. On the projects you've worked on, though, how is your team's flow for code integration, delivery, and deployment? What is your involvement on the configuration of these processes?

**Comment by MarcelLZ** [comment]
Yeah, Travis is so pretty easy \o/ So, when I joined HBSiS, I started on a mature team, where there was already a CI/CD flow running. Recently I have changed from my original team to another one, and there we don't have a CI/CD yet. I come with this objective, to introduce these flows, and yes, I'll need to configure everything =P I don't know how for now because I didn't do it before, but I'll need to do that. Now, we are teaching about git, pull request, branches, and git-flow to everyone. We need to complete this step first to go to the next :D
